lib/internal/cluster/RoundRobinHandle.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
```

Comment:
I used https://github.com/nodejs/node/blob/master/lib/internal/streams/BufferList.js as a reference for naming files that hold a single data type.